Clarify aggregation challenge solution (#382)
datacarpentry/sql-ecology-lesson: episodes/02-sql-aggregation.md

## Solution

> This query counts the number of species records that contain each value of the `taxa` field and names that result `species_count`.

Reviewer: ```suggestion
This query counts the number of times each value (Bird, Rabbit, Reptile or Rodent) in the `taxa` field occurs, defining a new field named `taxa_count` to hold the result.
```

The `GROUP BY` clause means the query will create an aggregated table with one row for each taxa.
Only those `taxa` values that have more than ten records will be included because of the `HAVING` clause.
This filtering is applied _after_ grouping has been done.

```sql

> SELECT taxa, COUNT(*) AS species_count

Reviewer: ```suggestion
SELECT taxa, COUNT(*) AS taxa_count
```

FROM species
GROUP BY taxa

> HAVING n > 10;

Reviewer: ```suggestion
HAVING species_count > 10;
```
